freqtrade/optimize/hyperopt.py

```diff
@@ -295,6 +296,50 @@ def _format_explanation_string(results, total_epochs) -> str:
                 f"{results['results_explanation']} " +
                 f"Objective: {results['loss']:.5f}")
 
+    @staticmethod
+    def print_result_table(config: dict, results: list, total_epochs: int, highlight_best: bool,
+                           print_colorized: bool) -> None:
+        """
+        Log result table
+        """
+        if not results:
+            return
+
+        trials = json_normalize(results, max_level=1)
+        trials = trials[['is_best', 'current_epoch',
+                         'results_metrics.trade_count', 'results_metrics.avg_profit',
+                         'results_metrics.total_profit', 'results_metrics.profit',
+                         'results_metrics.duration', 'loss']]
+        trials.columns = ['Best', 'Epoch', 'Trades', 'Avg profit', 'Total profit',
+                          'Profit', 'Avg duration', 'Objective']
+
+        trials['Best'] = trials['Best'].apply(lambda x: '*' if x else '')
+        trials['Objective'] = trials['Objective'].astype(str)
+
+        if print_colorized:
+            for i in range(len(trials)):
+                if trials.loc[i]['Total profit'] > 0:
+                    trials.at[i, 'Best'] = Fore.GREEN + trials.loc[i]['Best']
+                    trials.at[i, 'Objective'] = "{}{}".format(trials.loc[i]['Objective'],
+                                                              Fore.RESET)
+                    if '*' in trials.loc[i]['Best'] and highlight_best:
+                        trials.at[i, 'Best'] = Style.BRIGHT + trials.loc[i]['Best']
+                        trials.at[i, 'Objective'] = "{}{}".format(trials.loc[i]['Objective'],
+                                                                  Style.RESET_ALL)
```

Comment:
I think that the lines dividing the columns should not be colorized; I'll take this one after this PR is merged, I think I know how to make it easily...

Reply:
I was working a lot around this coloring topic tbh. Yes it can be done but from what i can tell the only way is to loop thru and set color on each individual column..
I can do that in this PR but will be later this week i guess.. on a trip for a few days

Reply:
It's only coloring the values now. Might be a more "pretty" way to do it but did not see much of a impact in performance